frontend/pages/admin/pacta-version/[id].vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import { type PactaVersion, type PactaVersionChanges } from '@/openapi/generated/pacta'
+
+const router = useRouter()
+const { pactaClient } = useAPI()
+const { error: { withLoadingAndErrorHandling, handleOAPIError } } = useModal()
+const { fromParams } = useURLParams()
+
+const id = presentOrCheckURL(fromParams('id'))
+
+const prefix = 'admin/pacta-version/[id]'
+const persistedPactaVersion = useState<PactaVersion>('totally different???')
+const pactaVersion = useState<PactaVersion>(`${prefix}.pactaVersion`)
+
+const changes = computed<PactaVersionChanges>(() => {
+  const a = persistedPactaVersion.value
+  const b = pactaVersion.value
+  if (!a || !b) { return {} }
+  return {
+    ...(a.name !== b.name ? { name: b.name } : {}),
+    ...(a.description !== b.description ? { description: b.description } : {}),
+    ...(a.digest !== b.digest ? { digest: b.digest } : {})
+  }
+})
+const hasChanges = computed<boolean>(() => Object.keys(changes.value).length > 0)
+
+const markDefault = () => withLoadingAndErrorHandling(
+  () => pactaClient.markPactaVersionAsDefault(id)
+    .then(handleOAPIError)
+    .then(() => { pactaVersion.value.isDefault = true }),
+  `${prefix}.markPactaVersionAsDefault`
+)
+const deletePV = () => withLoadingAndErrorHandling(
+  () => pactaClient.deletePactaVersion(id)
+    .then(handleOAPIError)
+    .then(() => router.push('/admin/pacta-version')),
+  `${prefix}.deletePactaVersion`
+)
+const saveChanges = () => withLoadingAndErrorHandling(
+  () => pactaClient.updatePactaVersion(id, changes.value)
+    .then(handleOAPIError)
+    .then(() => { persistedPactaVersion.value = pactaVersion.value })
+    .then(() => router.push('/admin/pacta-version')),
+  `${prefix}.saveChanges`
+)
+
+// TODO(#13) Remove this from the on-mounted hook
```

Comment:
Is the idea that SSR requests aren't currently working? If so, you'll want to update either the `useAPI` composable or underlying plugin to forward the `cookie` header, more or less like we do for AS: https://github.com/Silicon-Ally/AdventureScientists/blob/2b36eafd856949c1e0eaec48c5c07a4004aff092/frontend/plugins/graphql.ts#L62-L68

You'd set it using this: https://github.com/RMI-PACTA/app/blob/7e9e07aacec85441638b33249e75e0481444c54d/frontend/openapi/generated/pacta/PACTAClient.ts#L28

Reply:
Yep - that's essentially what I assume is happening. I've copied this suggestion into the bug, and I'll be following up shortly.